content/probability/README.md

```diff
@@ -3,6 +3,9 @@
 Incharge: Kiran Vasudev
```

Comment:
I'm not sure what "different view" means here...

I think we should also add another point to this session, namely "Introduction to Bayesian reasoning." The rest looks good.